### DISCUSSION TITLE
What is the decimal parameter?

### DISCUSSION THREAD

**Discussion by metrox-eth** [snapshot-labs/snapshot#773] [Q&A] [answered]
Suggest answers below or vote for an answer.

**Comment by metrox-eth** [comment, marked as answer]
- When you create a token you have to define the number of decimal that you want
- Most of the token use 18 decimals
- To find out how many decimals your token has you can search the contract address on etherscan.io

**Reply by Nguyendinhtuan17** [reply]
0xdfb4ACe028FB7C454e6785D16ad455edad5be94A

**Reply by jack36023602** [reply]
wow

**Comment by Vipalone** [comment]
When you create a token you have to define the number of decimal that you want
Most of the token use 18 decimals
To find out how many decimals your token has you can search the contract address on etherscan.io
